python/networkx.cpp

```diff
@@ -306,14 +306,14 @@ void add_key_value( //
     auto numeric_array = std::static_pointer_cast<array_type_at>(array);
 
     std::string jsons_str;
-    jsons_str.reserve(es.size() * (column_name.size() + 2));
+    jsons_str.reserve(es.size() * (sizeof(array_type_at) + column_name.size() + 2));
```

Comment:
The `sizeof(array_type_at)` part doesn't make sense. The printed length of a number is generally different from its binary length, especially for 64-bit values. We have a constant that defines an upper bound on it's printed length.